src/commands/connections/pool/set.ts

```diff
@@ -0,0 +1,21 @@
+import { runCommand } from '../../../util/command';
+import { outputFormat } from '../util';
+const open = require('open')
+
+export = {
+	command: 'set <id> [poolId] [poolName]',
```

Comment:
i suggest using add or assign instead of set.  also, i would think you'd only want to add by specifying the id of the pool instead of the name, here.  can a connection be  a part of more than one pool?

Reply:
if you specify by name only, it will create a new pool with that name (if one doesn't exist). i believe that's the only way to create a new pool right now

Reply:
I like `assign` and `unassign` since we are assigning a connection to a new/existing pool

Reply:
I would change the `[poolId] [poolName]` options into explicit options so you can put in descriptions for them. I would also recommend changing the option names to `pool: existing pool ID` and `name: new pool name`